.vscode/extensions.json

```diff
@@ -0,0 +1,8 @@
+{
+  "recommendations": [
+    "dbaeumer.vscode-eslint",
+    "EditorConfig.editorconfig",
+    "esbenp.prettier-vscode",
+    "stylelint.vscode-stylelint"
+  ]
+}
```

Comment:
let me know if these extension recommendations are helpful or just noisy

Reply:
i think they're quite helpful, we could even at some point later include an example-settings file